inst/chunks/fun_scale_time_err.stan

```diff
@@ -13,8 +14,48 @@
                          int[] nobs, int[] begin, int[] end) {
      vector[rows(zerr)] err;
      for (i in 1:size(nobs)) {
-       err[begin[i]:end[i]] =
-         sderr * chol_cor[1:nobs[i], 1:nobs[i]] * zerr[begin[i]:end[i]];
+       matrix[nobs[i], nobs[i]] L_i;
+       L_i = sderr * chol_cor[1:nobs[i], 1:nobs[i]];
```

Comment:
don't we need to recalculate the cholesky here if we subset??

Reply:
To my understanding, taking the first J row/col subset of a cholesksy factor is valid. Or am I mistaken?

Reply:
I see. Yes, that should work.